src/components/Answers.tsx

```diff
@@ -51,17 +63,21 @@ const Answers = ({
         return (
           <div
             key={answer.id + '_' + quizId + '_' + attempts}
-            className={`${styles.answers} ${indicatorClass}`}
+            className={`${styles.answers} ${size === 's' ? styles.answersSmall : ''} ${indicatorClass}`}
           >
             <input
-              id={answer.id}
+              id={`AI:${answer.id}-QI:${quizId}`}
               type={type === QuestionTypes.SingleChoice ? 'radio' : 'checkbox'}
               className={styles.input}
               name={'answer'}
```

Comment:
I think this causes a bug when there are several groups of radio buttons in the results table. There is only one `name` in the radio group so only one can be selected. So the user selection is no longer clear in the result.

![Screenshot 2024-04-26 at 10 43 40](https://github.com/openHPI/quiz-recap/assets/22977799/40ba3740-e905-48c5-a45a-3043d57789ef)

Reply:
here I had `name={`answer-${quizId}`}` and right before opening the PR I dropped this change because I didn't remember why it was needed... now I do! I'll put this back. Thanks!